### ISSUE TITLE
Create a live demo to show awesomeness

### ISSUE THREAD

**Issue by beatgammit** [beatgammit/tar-js#6]
Make a live demo with a simple form where users can enter a URL or text and tar it together.

Another cool idea is to do something like [this guy](http://www.williammalone.com/articles/create-html5-canvas-javascript-drawing-app/#demo-simple) does, except downloadable.

**Comment by sapioit** [comment]
Yeah, I was just about to ask for a demo (since I have no idea how to actually get it working in my Electron app), but I see someone already did this... more than 4 freaking years ago... (4 years, 5 months and 10 days ago, to be more accurate)....
